theme_avantgarde/static/src/scss/primary_variables.scss

```diff
@@ -177,7 +177,6 @@ $o-theme-font-configs: (
 $o-website-values-palettes: (
     (
         'color-palettes-name': 'avantgarde-3',
-        'layout': 'full',
```

Comment:
Don't hesitate to make extra PR that I can merge directly for things like these: defining layout = full in a theme is useless since this is a default (this is not wrong but I'd say it is best not to write it if useless).

If I can merge small stuff like these that are independent of your work directly, it will make the review easier later :+1: